### PR TITLE
Updating list of channels

### PR DESCRIPTION
**Pull request by vtamara**

### CODE REVIEW

**Comment by raisedadead** [comment]
Thanks @vtamara Looks good to me. @QuincyLarson and @SaintPeter @Rafase282 Please have a look and merge.

**Comment by Rafase282** [comment]
@hallaathrad is the camper sites's master.

**Comment by hallaathrad** [comment]
Made a few notes on the changes.

**Comment by SaintPeter** [comment]
Is there a reason you have not used Markdown link formatting?

IE: 
`French [FreeCodeCamp/TranslateFrench](https://gitter.im/FreeCodeCamp/TranslationFrench)`
displays as:
French [FreeCodeCamp/TranslateFrench](https://gitter.im/FreeCodeCamp/TranslationFrench)

**Comment by hallaathrad** [comment]
Lol, I totally oversaw that detail. @SaintPeter

**Comment by Rafase282** [comment]
Let me know when the changes has been made.

File: What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md [hunk 1]
```diff
@@ -4,22 +4,28 @@ We won't be able to add new languages to Free Code Camp until all of our
...
 * Brazilian Portuguese: https://gitter.im/FreeCodeCamp/BrazilianPortuguese
-* Korean: https://gitter.im/StephenMayeux/koreantranslations
-* Spanish: https://gitter.im/AlexTrapp/FreeCodeCampEspa%C3%B1ol
+* French https://gitter.im/FreeCodeCamp/TranslationFrench
+* German https://gitter.im/FreeCodeCamp/TranslationDeutsch
+* Hindi https://gitter.im/FreeCodeCamp/TranslateHindi
+* Korean https://gitter.im/StephenMayeux/koreantranslations
+* Russian https://gitter.im/FreeCodeCamp/TranslationRussian
+* Serbian https://gitter.im/FreeCodeCamp/TranslateSerbian
+* Spanish https://gitter.im/AlexTrapp/FreeCodeCampEspa%C3%B1ol
```
Comment (abhisekp) [inline What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md]: `https://gitter.im/FreeCodeCamp/Espanol` works

Comment (vtamara) [inline What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md]: But that channel is for support not for translation

**Comment by vtamara** [comment]
Thansk for suggestions @SaintPeter , @hallaathrad

**Comment by vtamara** [comment]
Done @Rafase282

File: What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md [hunk 2]
```diff
@@ -4,22 +4,33 @@ We won't be able to add new languages to Free Code Camp until all of our
...
+[Translators channel](https://gitter.im/FreeCodeCamp/Translators), 
+or create one yourself and share it at the  
+[translators channel](https://gitter.im/FreeCodeCamp/Translators) 
+and at other places to invite more translators and reviewers that speak your 
+language.  
 
-We invite you to read some recommendations based on the spanish team
-experience:
+We invite you to read some recommendations based on the spanish team experience:
   https://github.com/vtamara/fcc_trad/blob/master/Recomendaciones.EN.md
```
Comment (Rafase282) [inline What-to-do-if-you-speak-a-language-that-Free-Code-Camp-does-not-yet-support.md]: [Recommendations](https://github.com/vtamara/fcc_trad/blob/master/Recomendaciones.EN.md) or something like that would be nicer.

**Pull request merged**